server/vulnerabilities/nvd/sync.go

```diff
@@ -283,6 +283,43 @@ func LoadCVEMeta(ctx context.Context, logger log.Logger, vulnPath string, ds fle
 		metaMap[cve] = meta
 	}
 
+	return nil
+}
+
+func CVEMetaFromFiles(vulnPath string, logger log.Logger) (map[string]fleet.CVEMeta, error) {
+	metaMap := make(map[string]fleet.CVEMeta)
+
+	err := CVEMetaFromNVDFeedFiles(metaMap, vulnPath, logger)
+	if err != nil {
+		return nil, fmt.Errorf("nvd meta: %w", err)
+	}
+
+	err = CVEMetaFromEPSSFeedFiles(metaMap, vulnPath, logger)
+	if err != nil {
+		return nil, fmt.Errorf("epss meta: %w", err)
+	}
+
+	err = CVEMetaFromCISAFeedFiles(metaMap, vulnPath, logger)
+	if err != nil {
+		return nil, fmt.Errorf("cisa meta: %w", err)
+	}
+
+	return metaMap, nil
+}
```

Comment:
This code was written in a way that each progressive step adds more values to the metaMap struct and is dependent on the previous for correctness. It makes testing these methods independently very tricky.